### PR TITLE
[Doc] Use pluggable function runtime setting in the doc

### PR DESCRIPTION
**Pull request by RobertIndie**
### Motivation
In #5463, we have added the pluggable function runtime setting since 2.5.0. It is better if we can use this feature in the doc.

### Modifications

Update the doc to use pluggable function runtime.

### CODE REVIEW

**Review by Huanli-Meng — APPROVED** [review]
LGTM. Let's wait for a technical review from other engs.

**Review by jiazhai — APPROVED** [review]

**Review by eolivelli — APPROVED** [review]

**Review by sijie — APPROVED** [review]

**Pull request merged**
